src/gui/mainwindow.cpp

```diff
@@ -189,11 +189,17 @@ MainWindow::MainWindow(IGUIApplication *app, WindowState initialState)
 
     // Name filter
     m_searchFilter = new LineEdit(this);
-    m_searchFilter->setPlaceholderText(tr("Filter torrent names..."));
+    m_searchFilter->setPlaceholderText(tr("Filter torrents..."));
     m_searchFilter->setFixedWidth(200);
     m_searchFilter->setContextMenuPolicy(Qt::CustomContextMenu);
     connect(m_searchFilter, &QWidget::customContextMenuRequested, this, &MainWindow::showFilterContextMenu);
     m_searchFilterAction = m_ui->toolBar->insertWidget(m_ui->actionLock, m_searchFilter);
+    // Filter by type
+    m_filterBy = new QComboBox(this);
+    m_filterBy->addItem(tr("Torrent names"), TransferListModel::Column::TR_NAME);
```

Comment:
Looking at the GUI now and singular doesn't seem right to me... I don't know which one is correct.

Reply:
> Looking at the GUI now and singular doesn't seem right to me... I don't know which one is correct.

When I suggested singular I was under assumption that there is something like "Filter by" prefix.

Reply:
> When I suggested singular I was under assumption that there is something like "Filter by" prefix.

Does something like this would be good ?

![Capture d’écran du 2023-02-24 18-55-54](https://user-images.githubusercontent.com/38819112/221254577-82bcdb7d-a620-4218-b319-aa7d9af946a0.png)

Reply:
>I was under assumption that there is something like "Filter by" prefix.

Me too... anyway it seems the "Torrent" part threw me off or maybe both are correct.

>Does something like this would be good ?

I like it but no space before colon.

Reply:
> no space before colon.

👍